LICENSE

```diff
@@ -1,5 +1,5 @@
 
-                                 Apache License
+                                 No License
                            Version 2.0, January 2004
                         http://www.apache.org/licenses/
```

Comment:
Stills linking to Apache, I suggest updating the license to [GLWTPL](https://github.com/me-shaon/GLWTPL)